plugins/catalog-react/src/components/EntityPeekAheadPopover/CardActionComponents/EmailCardAction.tsx

```diff
@@ -23,14 +24,13 @@ import { Link } from '@backstage/core-components';
  *
  * @private
  */
-export const EmailCardAction = ({ email }: { email: string }) => {
+export const EmailCardAction = (props: { email: string }) => {
   return (
     <IconButton
       component={Link}
       aria-label="Email"
-      title={`Email ${email}`}
-      to={`mailto:${email}`}
-      target="_blank"
```

Comment:
Depending on how your email client is bound, you want to not make this a _blank because it just opens a wasteful empty page replacing the one you were on. In any case it should be up to Link to decide.